en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -1,24 +1,17 @@
-# Project Design Diagram
```

Comment:
The purpose of these changes was also to add more meaning to the ToC and main topics. Trying to answer the following questions better.

- How to open the Project Design view
- Exploring the Project Design view